src/bp/core/config/config-writer.ts

```diff
@@ -28,11 +33,22 @@ export class BotConfigWriter {
     const resourceLoader = new ModuleResourceLoader(this.logger, template.moduleId!)
     const templatePath = await resourceLoader.getBotTemplatePath(template.id)
     const templateConfig = path.resolve(templatePath, this.BOT_CONFIG_FILENAME)
-    const botDestinationPath = this.getBotFilePath(bot.id)
+    const botDestinationPath = path.join(process.PROJECT_LOCATION, `./data/bots/${bot.id}/`)
```

Comment:
That's not an issue at all because what you did will work but just so you know, you can ignore `./` when you use `path.join`. Both will work.